### PR TITLE
Issue-1336: DynamoDB Sink - use enum for available operations

### PR DESCRIPTION
**Pull request by manstis**
See https://github.com/apache/camel-kamelets/issues/1336

### CODE REVIEW

**Comment by manstis** [comment]
The `Build / build` and `Java Tests` steps fail as an additional Kamelet for `aws-ddb-experimental-sink` was generated.

I can either remove this _experimental_ Kamelet from the PR (as it does not exist on `main`) or should the test be updated? 

I didn't manually add the Kamelet it was generated (presumably missed/skipped previously?)

**Comment by oscerd** [comment]
The experimental kamelets has been removed already. Please rebase

File: kamelets/aws-ddb-sink.kamelet.yaml [hunk 1]
```diff
@@ -73,6 +73,7 @@ spec:
         title: Operation
         description: "The operation to perform. The options are PutItem, UpdateItem, or DeleteItem."
```
Comment (oscerd) [inline kamelets/aws-ddb-sink.kamelet.yaml:74]: In the documentation this will appear as enum, so we could remove them from the description

Comment (oscerd) [inline kamelets/aws-ddb-sink.kamelet.yaml:74]: In the documentation this will appear as enum, so we could remove them from the description

**Review by oscerd — CHANGES_REQUESTED** [review]
Just a minor finding

File: kamelets/aws-ddb-sink.kamelet.yaml (hunk 1 above)
Comment (manstis) [reply kamelets/aws-ddb-sink.kamelet.yaml]: Done.

Comment (manstis) [reply kamelets/aws-ddb-sink.kamelet.yaml]: Done. Thank-you for highlighting.

**Review by oscerd — APPROVED** [review]

**Comment by oscerd** [comment]
Thanks for this.

**Comment by oscerd** [comment]
I do think this should be backported on 3.x and 3.20.x too, but mind that there is the experimental one there. Thanks.

**Comment by manstis** [comment]
@oscerd 

> I do think this should be backported on 3.x and 3.20.x too, but mind that there is the experimental one there. Thanks.
- https://github.com/apache/camel-kamelets/pull/1339
- https://github.com/apache/camel-kamelets/pull/1340

**Pull request merged**